src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+
+    - `plain` (default): turn off traffic encryption,
+    - `ssl`: encrypt traffic by using the TLS 1.2 protocol (EE only).
+]])
+
+-- }}} <uri>.params configuration
+
+-- {{{ app configuration
+
+M['app'] = format_text([[
+    Using Tarantool as an application server, you can run your own Lua
+    applications. In the `app` section, you can load the application and
+    provide an application configuration in the `app.cfg` section.
+]])
+
+M['app.cfg'] = format_text([[
+    A configuration of the application loaded using `app.file` or `app.module`.
+]])
+
+M['app.cfg.*'] = format_text([[
+    Specify `file` (a path to a Lua file to load an application from) or
+    `module` (a Lua module to load an application from).
+]])
+
+M['app.file'] = 'A path to a Lua file to load an application from.'
+
+M['app.module'] = 'A Lua module to load an application from.'
+
+-- }}} app configuration
+
+-- {{{ audit_log configuration
+
+M['audit_log'] = format_text([[
+    The `audit_log` section defines configuration parameters related to
+    audit logging.
+]])
+
+M['audit_log.extract_key'] = format_text([[
+    If set to `true`, the audit subsystem extracts and prints only the primary
+    key instead of full tuples in DML events (`space_insert`, `space_replace`,
+    `space_delete`). Otherwise, full tuples are logged. The option may be
+    useful in case tuples are big.
+]])
+
+M['audit_log.file'] = format_text([[
+    Specify a file for the audit log destination. You can set the `file` type
+    using the audit_log.to option. If you write logs to a file, Tarantool
+    reopens the audit log at SIGHUP.
+]])
+
+M['audit_log.filter'] = format_text([[
+    Enable logging for a specified subset of audit events. This option accepts
+    the following values:
+
+    - Event names (for example, password_change). For details, see Audit log
+    events.
+    - Event groups (for example, audit). For details, see Event groups.
+
+    The option contains either one value from Possible values section or a
+    combination of them.
+
+    To enable custom audit log events, specify the custom value in this option.
+]])
+
+M['audit_log.filter.*'] = format_text([[
+    Specify a subset of audit events to log by providing a value
+    from the allowed list of events or groups.
+]])
+
+M['audit_log.format'] = 'Specify a format that is used for the audit log.'
+
+M['audit_log.nonblock'] = format_text([[
+    Specify the logging behavior if the system is not ready to write.
+    If set to `true`, Tarantool does not block during logging if the system
+    is non-writable and writes a message instead. Using this value may
+    improve logging performance at the cost of losing some log messages.
+]])
+
+M['audit_log.pipe'] = format_text([[
+    Specify a pipe for the audit log destination. You can set the `pipe` type
+    using the audit_log.to option. If log is a program, its pid is stored in
+    the `audit.pid` field. You need to send it a signal to rotate logs.
+]])
```

Comment:
'Specify a pipe' has unclear meaning. Should I specify a file descriptor or something else?

In fact, a command is written here. See the `log.pipe` description for example.

Reply:
Maybe like this?
```lua
I['audit_log.pipe'] = format_text([[
    Start a program and write logs to its standard input (`stdin`). To send
    logs to a program's standard input, you need to set `audit_log.to` to
    `pipe`. If log is a program, its pid is stored in the `audit.pid` field.
]])
```

Reply:
> If log is a program

I guess it is related to the `box.cfg()` option, where the same option may refer to a program (using the `pipe:` or `|` prefix), to syslog (`syslog:`), to file (`file:`).

I would say:

> If the pipe logging is configured

or just

> [Start a program and write logs to its standard input (`stdin`).] The PID of the program is stored in the `audit.pid` field. To send logs <...>

or (if we want to repeat the same words less)

> [Start a program and write logs to its standard input (`stdin`).] The PID of the child process is stored in the `audit.pid` field. To send logs <...>

Reply:
"If log is a program, its pid is stored in the `audit.pid` field." dropped.